src/main/java/com/aerospike/restclient/domain/querymodels/QueryFilter.java

```diff
@@ -45,29 +45,28 @@
         }
 )
 @Schema(
-        description = "QueryFilter base type. Only allowed on bin which has a secondary index defined.", oneOf = {
-        QueryEqualsStringFilter.class,
-        QueryEqualLongFilter.class,
-        QueryRangeFilter.class,
-        QueryContainsStringFilter.class,
-        QueryContainsLongFilter.class,
-        QueryGeoWithinPolygonFilter.class,
-        QueryGeoWithinRadiusFilter.class,
-        QueryGeoContainsPointFilter.class,
-},
+        description = "QueryFilter base type. Only allowed on bin which has a secondary index defined.",
+        oneOf = {
+                QueryEqualsStringFilter.class,
+                QueryEqualLongFilter.class,
+                QueryRangeFilter.class,
+                QueryContainsStringFilter.class,
+                QueryContainsLongFilter.class,
+                QueryGeoWithinPolygonFilter.class,
+                QueryGeoWithinRadiusFilter.class,
+                QueryGeoContainsPointFilter.class,
+        },
         externalDocs = @ExternalDocumentation(url = "https://javadoc.io/doc/com.aerospike/aerospike-client/6.1.2/com/aerospike/client/query/Filter.html")
```

Comment:
make the domain with the version part path a constant

Reply:
Yes, forgot to do this